example/src/App.tsx

```diff
@@ -1,11 +1,12 @@
 import dedent from "@cometlib/dedent";
 import {
-  Align,
   AttachStep,
-  Position,
   SpotlightTourProvider,
   TourBox,
   TourStep,
+  flip,
+  offset,
+  shift,
```

Comment:
@alejo0o After looking at the example, my line of thought here is that `react-native-spotlight-tour` is exporting components but maybe something like `react-native-spotlight-tour/floating-middlewares` should import those other operations `flip, offset, shift` and so on, let me know what do you think

Reply:
Hey @well1791 we talked this with @JoseLion and we would rather have those dependencies under spotlight tour since we avoid creating to much import statements in one file

Reply:
@well1791 another option would be to group the middleware exports in an object and then only export that object, like some sort of enum:
```ts
import { flip, offset, shift } from "@floating-ui/react-native";

export const Middleware = {
  flip,
  offset,
  shift,
};

// ...later on userland

import { Middleware } from "@stackbuilders/react-native-spotlight-tour";

floatingProps:{
  middleware: [Middleware.offset(0), Middleware.shift(), Middleware.flip()],
  placement: "right",
}
```

It's more verbose upon usage IMO, but what do you think? 🤔

Reply:
@alejo0o the decision makes total sense, also @JoseLion moving it into its own object looks great! Agreed on it being verbose but still, this small improvement makes a point on usability. Think of this (although not realistic enough), there may be a case where a dev want to build a lib based on this and may want to do something like this.

```tsx
import { Middleware } from "@stackbuilders/react-native-spotlight-tour";

// ...
type RandomObject = {
  validMiddleware: keyof typeof Middleware

```

If I were the one needed that, I would appreciate having this in advance ^^

Reply:
I agree! We can provide both so people can choose the best way for them. I think we can tackle this on a separate PR thought. Thanks for your inputs, it's always helpful to hear different perspectives 🙂